docs/tutorial/part-zero/index.md

```diff
@@ -67,43 +67,20 @@ The steps to download and install Git depend on your operating system. Follow th
 
 > 💡 You will not need to know Git to complete this tutorial, but it is a very useful tool. If you are interested in learning more about version control, Git, and GitHub, check out GitHub's [Git Handbook](https://guides.github.com/introduction/git-handbook/).
 
-## Install Gatsby CLI
+## Using the Gatsby CLI
 
-The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run commands for developing Gatsby sites. It is a published npm package. You can install Gatsby CLI from the npm registry, using the npm CLI.
+The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run commands for developing Gatsby sites. It is a published npm package. Rather than installing the Gatsby CLI tool locally you can run it using [npx](https://www.npmjs.com/package/npx)
 
-1.  Navigate to the terminal.
-2.  Run `npm install --global gatsby-cli`.
-
-> 💡 If you are unable to successfully install due to a permissions issue, you may want to check out the [npm docs on fixing permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions), or [this guide](https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md).
-
-A couple of different things are happening here.
-
-```shell
-npm install --global gatsby-cli
-```
-
-- You're using the npm CLI to install Gatsby CLI. `npm install` is the command used to install packages.
-- When installing npm packages, you can install them globally or in a specific project. (You’ll learn about the latter, later). The `--global` flag signals that we want the first option to install globally. This means your package will be available to you on your computer, outside of the context of a specific project.
-- `gatsby-cli` is the exact name our desired package is registered with on the [**npm registry**](https://www.npmjs.com/package/gatsby-cli).
-
-### Check your Gatsby CLI installation
-
-1.  Open up your terminal.
-2.  Run `gatsby --version`.
-3.  Run `gatsby --help`.
-
-![Check gatsby version in terminal](02-gatsby-version.png)
```

Comment:
I think we'll want to keep these validation steps, or at least provide some way to illustrate that something worked!

Reply:
Since we aren't installing the CLI globally these verifications won't do anything.

Maybe after the project is generated we could verify the gatsby version in the project dir?

3.  Run `cd hello-world`.
+ Run `npx gatsby --version` .  <--- something like this?
4.  Run `gatsby develop`.
4.  Run `npm run develop`.

It might be overkill though? If `npx gatsby new project-name` doesn't work then the project won't be generated which should be fairly obvious.